pkgs/development/interpreters/janet/default.nix

```
@@ -29,11 +36,26 @@ stdenv.mkDerivation rec {
     $out/bin/janet -e '(+ 1 2 3)'
   '';
 
```

```suggestion
  # Tests require binding to local addresses for listening sockets.
  __darwinAllowLocalNetworking = true;
```

This should fix the darwin test failure

```
Starting suite /private/tmp/nix-build-janet-1.32.0.drv-0/source/build/../test/suite-ev.janet...
error: Socket is not connected
  in net/write [src/core/net.c] on line 893
  in <anonymous> [/private/tmp/nix-build-janet-1.32.0.drv-0/source/build/../test/suite-ev.janet] on line 200, column 7
  in test-echo [/private/tmp/nix-build-janet-1.32.0.drv-0/source/build/../test/suite-ev.janet] on line 199, column 5
  in _while [/private/tmp/nix-build-janet-1.32.0.drv-0/source/build/../test/suite-ev.janet] on line 204, column 3
  in _thunk [/private/tmp/nix-build-janet-1.32.0.drv-0/source/build/../test/suite-ev.janet] (tailcall) on line 183, column 1
```

---

it didn't work it seems, not sure how to proceed

---

Let's disable that test on darwin 

Removing this line https://github.com/janet-lang/janet/blob/cc5beda0d26c0b91fb28540291e950392cf59fc4/meson.build#L246 should work, There's no other mention of `suite-ev.janet` in the repo.